tools/mc-java-protoc/src/main/java/io/spine/tools/mc/java/protoc/CodeGenerator.java

```diff
@@ -222,7 +222,7 @@ private void checkCompilerVersion(CodeGeneratorRequest request) {
 
     private static File concatContent(File left, File right) {
         return left.toBuilder()
-                   .setContent(left.getContent() + right.getContent())
+                   .setContent(left.getContent() + System.lineSeparator() + right.getContent())
```

Comment:
While I realise this is a correct version, it as well looks a breaking change to me. 

Please consider renaming the method, or at least documenting this behaviour.